suse_openstack_cloud

```diff
@@ -396,13 +396,12 @@ fi
 section_header "RabbitMQ broker"
 
 if [ -e /usr/sbin/rabbitmqctl ]; then
-    rabbitmq_tmpdir=$(mktemp -d)
-    rabbitmq_report_log=$rabbitmq_tmpdir/rabbitmq_report.log
+    rabbitmq_var_log="/var/log/rabbitmq"
+    rabbitmq_report_log=$rabbitmq_var_log/rabbitmq_report.log
```

Comment:
It will be not logrotated because it will be removed after supportconfig finish, but I know what do you mean. As you wish I can remove suffix for this file